resources/south-america/brazil/mapeadoresYouthMappersUFRJ.json

```diff
@@ -0,0 +1,62 @@
+{
+  "id": "OSM-group",
+  "type": "group",
+  "locationSet": {
+  
+    "include": ["076"];
```

Comment:
this ; looks suspect, marked on github interface, seems confirmed by https://duckduckgo.com/?t=ffab&q=json+validator&ia=answer
```
Error: Parse error on line 6:
...  "include": ["076"];    "include": ["0
-----------------------^
Expecting 'EOF', '}', ',', ']', got 'undefined'
```